.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
```

Should be master branch..

```suggestion
      - master
```